### PR TITLE
gh-95471: IDLE - Tweak Edit menu

### PR DESCRIPTION
**Pull request by terryjreedy**
Move `Select All` above  `Cut` as it is used with `Cut` and `Copy`
but not `Paste`.  Add a separator between `Replace` and
`Go to Line` to show IDLE issue triagers the separation between
'Edit-find' and 'Edit-show' topics.

* Issue: gh-95471

### CODE REVIEW

**Comment by terryjreedy** [comment]
@erlend-aasland Do you think the blurb is too wordy?  If so, the last two sentences could be
```
Move ``Select All`` and add a new separator.
```

**Comment by erlend-aasland** [comment]
> @erlend-aasland Do you think the blurb is too wordy? If so, the last two sentences could be
> 
> ```
> Move ``Select All`` and add a new separator.
> ```

IMHO, yes. The changelog is huge; keeping each item succinct is nice. Maybe prefix with `Edit menu: ` or `In Edit menu,`?

**Comment by terryjreedy** [comment]
Good point.  Unlike NEWS.txt, blurbs have a direct link to the issue.  In this case, I copied the expanded text from the initial issue message.  We can now edit such when an issue is closed to give a good summary for anyone visiting.
